BWE8LT/BWE8LT/Application.cs

```diff
@@ -0,0 +1,52 @@
+using BWE8LT.Controller;
+using BWE8LT.Controller.Implementations;
+using BWE8LT.Services;
+using BWE8LT.Services.Implementations;
+
+namespace BWE8LT;
+
+public static class Application
+{
+    public static void Start(string currentWorkingDirectory)
+    {
+        ICommandService commandService = new CommandService();
+        IConsoleController consoleController = new ConsoleController(currentWorkingDirectory, commandService);
+        try
+        {
+            consoleController.CommandService.LoadCommands(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Config", "Commands.json"));
+        }
+        catch (DirectoryNotFoundException ex)
+        {
+            Console.Error.WriteLine(ex.Message);
+            Environment.Exit(-1);
+        }
+        catch (ArgumentException ex)
+        {
+            Console.Error.WriteLine(ex.Message);
+            Environment.Exit(-1);
+        }
+        
+        Console.Clear();
+        
+        consoleController.CurrentWindow.SetCursorVisibility(false);
+        consoleController.CurrentWindow.FileService.ReadAllFiles(
+            consoleController.CurrentWindow.FileService.WorkingDirectory
+        );
+        
+        consoleController.CurrentWindow.UpdateHeader([
+            "Current working directory:", 
+            consoleController.CurrentWindow.FileService.WorkingDirectory
+        ]);
+        consoleController.CurrentWindow.WriteLoadedFilesToConsole();
+        consoleController.CurrentWindow.UpdateFooter([consoleController.GetWindowIndicators()]);
+        
+        consoleController.CurrentWindow.Cursor.MoveCursor(0);
+        consoleController.CurrentWindow.RefreshDisplay();
+
+        while (true)
+        {
+            ConsoleKeyInfo pressedKey = consoleController.CommandService.ReadKeyCommand();
```

Comment:
Note: A ConsoleKeyInfo beolvasás valahol Console.ReadKey-t fog használni, ami ha át van irányítva a programod (pl Pipe operátorral), akkor keservesen el fog halni. Ez nem feltétlen baj, de lehet érdemes lenne detektálni a program indításakor valahogy (fogalmam sincs hogy hogy most hirtelen) és kilépni egy hibakóddal, hogy nem támogatott use case. Persze ezt csak akkor, ha ezt amúgy közzé is akarod tenni github-on a házi keretén kívül.